### ISSUE TITLE
Getting the client IP address

### ISSUE THREAD

**Issue by brbeaird** [FreeMasen/luxure#16]
Is there any way to get the client IP address, maybe something similar to req.ip in NodeJS? In my case, there is no reverse-proxy involved, so there is nothing like x-forwarded-for in the header.

**Comment by brbeaird** [comment]
I think I actually just found it - req.socket:getpeername()

**Comment by FreeMasen** [comment]
Yes, that would be the best way to get the peer name, it might be worth adding some kind of helper to the request type to make this a little more obvious
